### ISSUE TITLE
可以支持多卡部署并行推理么？

### ISSUE THREAD

**Issue by wangyiedge** [OpenBMB/OmniLMM#29]

**Comment by iceflame89** [comment]
Yes,  change here to `True`,   would support deploy on multi-GPUs with small memory size.

https://github.com/OpenBMB/OmniLMM/blob/2553592bae4034f57557a593c8609a0611919d10/chat.py#L30-L36
